integration-tests/mockApis/auth.js

```diff
@@ -16,10 +16,12 @@ const createToken = () => {
 }
 
 const getLoginUrl = () =>
-  getRequests().then(data => {
```

Comment:
getting all requests was a bit overkill.  Also found that some images were pending and browser got them again so therefore `requests` contained them at the top